releaseNotes/Microsoft.Featuremanagement.Telemetry.ApplicationInsights.AspNetCore.md

```diff
@@ -0,0 +1,19 @@
+# Microsoft.FeatureManagement.Telemetry.ApplicationInsights.AspNetCore
+
+[Source code][source_code] | [Package (NuGet)][package] | [Samples][samples] | [Product documentation][docs]
+
+## 4.0.0-preview2 - March 7, 2024
+
+### Enhancements
+
+* Added `TargetingTelemetryInitializer`, which appends targeting information to all outgoing telemetry to Application Insights. ([#409](https://github.com/microsoft/FeatureManagement-Dotnet/issues/409))
```

Comment:
Fine with me- I have a slight tweak since telemetry correlation was possible before:

Introduced a telemetry initializer named TargetingTelemetryInitializer. It automatically adds targeting information to telemetry data your application sends to Application Insights. *This can be used to* correlate your telemetry data with feature evaluation events based on the targeting information during your telemetry analysis.

Reply:
Sounds good.